examples/stress_tests/many_animated_sprites.rs

```diff
@@ -1,10 +1,9 @@
 //! Renders a lot of animated sprites to allow performance testing.
 //!
 //! It sets up many animated sprites in different sizes and rotations,
-//! and at different scales in the world, and moves the camera over them.
+//! and at different scales in the world,
+//! and moves the camera over them to see how well frustum culling works.
```

Comment:
Nit: The sentence as a whole here is

> It sets up many animated sprites in different sizes and rotations, and at different scales in the world, and moves the camera over them to see how well frustum culling works.

I think there are too many "and". What about the following?

> This example sets up many animated sprites in different sizes, rotations, and scales in the world. It also moves the camera over them to see how well frustum culling works.

Reply:
You are right, that does flow better, thank you